source/sass/mozfest.scss

```diff
@@ -19,6 +19,7 @@ body.mozfest {
 
   &#view-mozfest-home {
     #hero {
+      $breakpoint-larger: $bp-md;
```

Comment:
are these technically all the "same" breakpoint? There's a lot of declarations for this same-named var, I'm wondering if we can just add this to our list of global variables instead.

Reply:
Good point. Each page / component were designed individually. It's coincidental that some breakpoints happen to be the same. That's why I think these variables should stay in local scope instead being declared globally.